lib/ctes_in_my_pg.rb

```diff
@@ -1,6 +1,16 @@
 require "ctes_in_my_pg/version"
 #require 'byebug'
 
+module CtesInMyPg
+  class << self
+    def supports_materialization_specifiers?
```

Comment:
I prefer `self. supports_materialization_specifiers?` syntax, but it's not a big deal.